### PR TITLE
add link to badges

### PR DESCRIPTION
**Pull request by tuxtof**
**What this PR does / why we need it**:

add link to CI pages inside the README

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 ---
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/nutanix-cloud-native/cluster-api-provider-nutanix)](https://goreportcard.com/report/github.com/nutanix-cloud-native/cluster-api-provider-nutanix)
-![CI](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/actions/workflows/build-dev.yaml/badge.svg)
-![Release](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/actions/workflows/release.yaml/badge.svg)
+[![CI](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/actions/workflows/build-dev.yaml/badge.svg)](https://github.com/tuxtof/cluster-api-provider-nutanix/actions/workflows/build-dev.yaml)
```
Comment (deepakm-ntnx) [inline README.md:6]: tuxof => nutnaix-cloud-native?
eg: https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/actions/workflows/build-dev.yaml

File: README.md [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
+[![Release](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/actions/workflows/release.yaml/badge.svg)](https://github.com/tuxtof/cluster-api-provider-nutanix/actions/workflows/release.yaml)
```
Comment (deepakm-ntnx) [inline README.md:7]: same here? 
https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/actions/workflows/release.yaml

Comment (tuxtof) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (tuxtof) [reply README.md]: fixed

**Review by yannickstruyf3 — DISMISSED** [review]

**Review by thunderboltsid — APPROVED** [review]

**Review by yannickstruyf3 — APPROVED** [review]

**Pull request merged**